geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/tables/AttributeTable.scala

```diff
@@ -51,7 +51,7 @@ object AttributeTable extends GeoMesaTable with LazyLogging {
   override def supports(sft: SimpleFeatureType) =
     sft.getSchemaVersion > 5 && sft.getAttributeDescriptors.exists(_.isIndexed)
 
-  override val suffix: String = "attr_idx"
+  override val suffix: String = "attr"
```

Comment:
we do have a couple of places where we used that...  @jnh5y see Alec's email :cry: